schemas/group-attributes.v1.schema.json

```diff
@@ -21,6 +21,9 @@
         "substatus": {
           "type": ["integer", "null"]
         },
+        "priority": {
+          "type": ["integer", "null"]
```

Comment:
Does the null here also include not present?  Since we won't be including this column at all at first

Reply:
Not adding it to the `required` section at the bottom of the schema ensures that